Documentation/workflow/EditingAndDebugging.md

```diff
@@ -0,0 +1,50 @@
+
+# Editing and Debugging
+
+If you are editing on the windows Operating system, Using Visual Studio 2015 is a good option for editing
+the code in this repository.    You can of course also use the editor of your choice.   One further option
+is to use [VSCode](https://code.visualstudio.com/) which is a light weight, cross-platform that like
+Visual Studio, is optimized for development workflow (code editing and debugging) but works on more platforms
+(in particular Linux)
+
+[VSCode](https://code.visualstudio.com/) has built-in support for syntax highlightinng and previewing 
+markdown (*.md) files that GIT repositories like this one use for documentation.   If you want to modify
```

Comment:
Include `*.md` in backticks - * is treated as switch to italics otherwise by GitHub markdown renderer